packages/jest-config/index.js

```diff
@@ -9,12 +9,12 @@ module.exports = {
       require.resolve("babel-jest"),
```

Comment:
I've missed this change in the previous PR (https://github.com/webstudio-is/webstudio-designer/commit/47a7bd6e4baefe0351ca80b1985ca626121bed86#diff-835ed62c13e91354b349566613127e466f4f597630588cc90d9ba5c3b8f56ad0R9) - why esbuild has been replaced with Babel here? 🤔

Reply:
Because that esbuild plugin unconditionally run babel/plugin-transform-modules-commonjs

Reply:
It actually calls it conditionally:
https://github.com/aelbore/esbuild-jest/blob/daa5847b3b382d9ddf6cc26e60ad949d202c4461/src/index.ts#L28-L33

But ye, this is quite... something :o Was this creating any issues though?

Reply:
You are right. Didn't check format option.